test/e2e/storage/persistent_volumes-local.go

```diff
@@ -268,13 +285,35 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 			Context("Set fsGroup for local volume", func() {
 
 				It("should set fsGroup for one pod", func() {
+					if testVolType == BlockLocalVolumeType {
+						framework.Skipf("We don't set fsGroup on block device, skipped.")
```

Comment:
This could go in a beforeeach, since the all fsgroup tests should be skipped

Reply:
Done in https://github.com/kubernetes/kubernetes/pull/62903/commits/b505e92b9ffc3cd61f380639af4fa07d2f003030.